tgui/packages/tgui/interfaces/ShuttleConsole.js

```diff
@@ -96,7 +96,9 @@ const ShuttleConsoleContent = (props, context) => {
               ? 'average'
               : status==="Igniting"
                 ? 'average'
-                : 'bad'}
+                : status==="Recharging"
+                  ? 'average'
+                  : 'bad'}
```

Comment:
This is a nasty ternary. Can you split this out into a switch/case?

Reply:
Commited a much cleaner solution with a color keys instead, which allows to easily modify existing colors for the each status, or add a new one.

Reply:
Works for me!